flink_scrat/main.py

```diff
@@ -68,18 +86,31 @@ def parse_args():
     savepoint_parser.set_defaults(action="savepoint")
 
     args = parser.parse_args()
+    
+    if args.use_yarn == True and (args.app_name is None or args.yarn_address is None or args.yarn_port is None):
```

Comment:
Here you could use `if args.use_yarn and (not args.app_name or not args.yarn_address or not args.yarn_port):`